README.md

```diff
@@ -82,6 +82,15 @@ Default value: `[ 'http://google.com' ]`
 An array containing strings, that represents the wished urls for the photosession.
 
 
+#### options.relativePaths
+
+Type: `Boolean`
+
+Default value: `false`
+
+If final images should contain only realative urls.
+
+
 #### options.userName
```

Comment:
Just had a quick look - but I'm not sure I get it completely how this works. 

Maybe we can add another example at the end of the readme explaining a bit more? :bowtie: